### ISSUE TITLE
grass under trees mows itself, stays mowed

### ISSUE THREAD

**Issue by cheweytoo** [OpenRCT2/OpenRCT2#18517]
### Operating System

Ubuntu Linux 20.04.5

### OpenRCT2 build

3a9900c develop, AppImage

### Base game

RollerCoaster Tycoon 2

### Describe the issue

If a tree is planted on a tile with grown grass, that tile will revert to its default mown state after a while - and then stay there.
That makes for some weirdly clean looking forests.

Hyper Speed demonstration:

https://user-images.githubusercontent.com/29800630/200138607-b4b81fd7-7bc6-43de-a896-143cd79feaf7.mp4

### Steps to reproduce

- build tree on tile with grown grass
- wait a bit
- tile will mow itself, and never grow back

**Comment by cheweytoo** [comment]
Further observation: time-to-selfmow is variable. In some cases it is almost immediately, in others it takes up to three days.

**Comment by ocalhoun6** [comment]
Pretty sure it's normal for the grass underneath trees to be in the mowed state... It's just I thought that happened instantly. It's weird that there's a delay.

What's the Vanilla RCT2 behavior?

**Comment by cheweytoo** [comment]
Well, I'm proposing it shouldn't be ;-)

No idea about the vanilla behavior, sorry – I should really revive that Windows VM one of these days…

**Comment by duncanspumpkin** [comment]
Yes this is expected behaviour. There is a periodic update function that updates surfaces that will tidy up the surface when scenery is placed on them

**Comment by Gymnasiast** [comment]
The default state isn’t mown - when the grass is mown, you get lines. It’s a neutral state.

Anyway, this is not a bug in any sense. It’s intended behaviour and as such belongs in Discussions and not under issues. Moving it there.
